### DISCUSSION TITLE
Other related tools

### DISCUSSION THREAD

**Discussion by preinheimer** [perftools/xhgui#416] [General]
Hey @glensc there's a discussion https://github.com/code-lts/welcome/issues/6 about some other XHProf related tools. 

Since you're the one actively maintaining this one, I was wondering if you had thoughts on where https://github.com/preinheimer/xhprof should live? 

thanks for all your hard work

**Comment by glensc** [comment]
@preinheimer https://github.com/preinheimer/xhprof - it also contains extension, but the maintained extension is at https://github.com/longxinH/xhprof. I think you should get into agreement that one is for UI (renamed to xhprof-ui) and the other is extension (renamed to xhprof-extension), and drop the unrelated code from respective repositories.

As for the home for the projects, you can move both to @perftools organization, if you don't have access @markstory can organize.

For me, using language-agnostic name like "perf tools" was overly greedy name, like other languages don't exist, and doesn't the projects in, are usable for other languages.

But from https://github.com/code-lts/welcome/issues/6, I see there's also organization https://github.com/xhprof, perhaps they all belong there, and @perftools to be obliterated (archived)?

please do note that there are two archived projects in @pertools organization:
- https://github.com/perftools?q=&type=&language=
  - https://github.com/perftools/Inclued-UI
  - https://github.com/perftools/xhgui-collector

**Reply by glensc** [reply]
This is also an interesting organization:
- https://github.com/phperf?q=&type=source&language=

**Reply by glensc** [reply]
If the question is whether to move to @code-lts, then I guess it's a no, as their rules indicate the project needs to be dead first:
- https://github.com/code-lts/welcome/blob/f40ad0b920fc5ea52d24812397d9373d745ca228/README.md#the-rules
